deploy_reactors.py

```diff
@@ -32,9 +33,15 @@ def make_recipe(in_dict, in_template):
 
 
 def main(*args):
-    data = get_lifetime_and_name(*args)
-    input_temp = load_template('./templates/deploy_template.xml')
-    make_recipe(data, input_temp)
+    if args[0] == ['all']:
+        lists = []
+        for files in os.listdir('./cyclus_input/reactors'):
```

Comment:
These paths should be script commandline arguments. This path should not be hardcoded.